packages/client/src/api/wargames_api/index.ts

```diff
@@ -725,6 +738,43 @@ export const postNewMessage = async (dbName: string, details: MessageDetails, me
   })
 }
 
+/**
+ * Populate a new wargame with bulk data
+ * 
+ * @param dbName The name of the database to create the wargame in
+ * @param bulkData The bulk data to populate the wargame with
+ * 
+ * @returns Promise that resolves with the populated wargame
+ */
+export const postPopulateWargame = (dbName: string, bulkData: Array<Message | Wargame>): Promise<Wargame> => {
+  const wargameName = dbName.replace('.json', '')
+
+  // Generate a unique name for the wargame by appending a timestamp to the end of the name
+  const name: string = `${wargameName}-${uniqid.time()}`
+
+  // Create a new database provider instance for the new wargame
+  const db = new DbProvider(databasePath + name)
+  addWargameDbStore({ name: name, db })
+  const customBulkMessage = bulkData
+
+  // Return a new promise that will resolve with the populated wargame
+  return new Promise((resolve, reject) => {
+    // Call the bulkDocs() function of the new database instance, passing in the bulk data
+    db.bulkDocs(customBulkMessage).then(() => {
+      // Call getLatestWargameRevision() to retrieve the latest revision of the new wargame
+      getLatestWargameRevision(name).then((res) => {
+        // @ts-ignore
+        resolve(res)
+      }).catch((err) => {
+        reject(err)
```

Comment:
Shouldn't we log the above error?  Otherwise it looks like it will silently fail, and we'll miss out on the diagnostics.